app/assets/javascripts/app/views/header_view.js

```diff
@@ -7,36 +7,27 @@ app.views.Header = app.views.Base.extend({
   className : "dark-header",
 
   events : {
-    "click ul.dropdown li:first-child" : "toggleDropdown",
+    "click ul.nav.navbar-nav li:first-child" : "toggleDropdown",
     "focusin #q": "toggleSearchActive",
     "focusout #q": "toggleSearchActive"
   },
 
   initialize : function(options) {
-    $(document.body).click($.proxy(this.hideDropdown, this));
     return this;
   },
 
   menuElement : function() {
-    return this.$("ul.dropdown");
+    return this.$("ul.nav.navbar-nav");
```

Comment:
HI! I'm on vacation and the hostel has a very very slow wifi, but I'll love
to help if I find the time and bandwidth one of these nights!



On Thu, Jan 22, 2015 at 12:10 PM, Dumitru Ursu <notifications@github.com>
wrote:

> In app/assets/javascripts/app/views/header_view.js
> <https://github.com/diaspora/diaspora/pull/5480#discussion_r23381215>:
>
> >      return this;
> >    },
> >
> >    menuElement : function() {
> > -    return this.$("ul.dropdown");
> > +    return this.$("ul.nav.navbar-nav");
>
> This would be an excelent place of an ID. ID are wonderful for attaching
> JS events handlers and functions. The mark up will definitely change later,
> as we move to BS3.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diaspora/diaspora/pull/5480/files#r23381215>.
>